src/SFA.DAS.AANHub.Application/CalendarEvents/Queries/GetCalendarEventsQueryResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿using SFA.DAS.AANHub.Domain.Models;
+
+namespace SFA.DAS.AANHub.Application.CalendarEvents.Queries;
+public class GetCalendarEventsQueryResult
+{
+    public int Page { get; set; }
+    public int PageSize { get; set; }
+    public int TotalPages { get; set; }
+    public int TotalCount { get; set; }
```

Comment:
I thought you said you have removed all the paging bits

Reply:
There is still a written requirement to return a page, pagesize, totalpages, totalcount in the documentation

Reply:
yes but we will deliver that in the relevant story